src/Adapter/Product/Combination/CommandHandler/RemoveCombinationCommandHandler.php

```diff
@@ -34,6 +34,8 @@
 
 /**
  * Handles @see RemoveCombinationCommand using adapter udpater service
+ *
+ * todo: rename -> DeleteCombinationHandler
```

Comment:
Can I simply rename as this class was introduced in experimental page, or do I still have to copy & deprecate it?

Reply:
IMO yes